doc/man/8/mount.ceph.rst

```diff
@@ -23,170 +26,193 @@ client component does most of the real work. In fact, it is possible
 
-:command:`recover_session=<no|clean>`
-  Set auto reconnect mode in the case where the client is blacklisted. The
-  available modes are ``no`` and ``clean``. The default is ``no``.
+:command:`secretfile`
+    path to file containing the secret key to use with cephx
 
-  - ``no``: never attempt to reconnect when client detects that it has been blacklisted. Blacklisted clients will not attempt to reconnect and their operations will fail too.
+:command:`snapdirname`
+    string, set the name of the hidden snapdir. Default: .snap
 
-  - ``clean``: client reconnects to the ceph cluster automatically when it detects that it has been blacklisted. During reconnect, client drops dirty data/metadata, invalidates page caches and writable file handles.  After reconnect, file locks become stale because the MDS loses track of them. If an inode contains any stale file locks, read/write on the inode is not allowed until applications release all stale file locks.
+:command:`write_congestion_kb`
+    int (kb), max writeback in flight. scale with available
+    memory. Default: calculated from available memory
 
+:command:`wsize`
+    int (bytes), max write size. Default: 16777216 (16*1024*1024) (writeback
+    uses smaller of wsize and stripe unit)
 
-:command: `mds_namespace=<fs-name>`
-    Specify the non-default file system to be mounted. Not passing this option
-    mounts the default file system.
 
 Mount Secrets
 =============
 If the `secret` and `secretfile` options are not specified on the command-line
 then the mount helper will spawn a child process that will use the standard
 ceph library routines to find a keyring and fetch the secret from it.
 
+
 Examples
 ========
 
 Mount the full file system::
 
-        mount.ceph monhost:/ /mnt/foo
+    mount.ceph :/ /mnt/mycephfs
 
-If there are multiple monitors::
+When authentication is enabled on Ceph cluster::
 
-        mount.ceph monhost1,monhost2,monhost3:/ /mnt/foo
+    mount.ceph :/ /mnt/mycephfs -o name=fs
+
+You can pass Monitor host's IP address yourself too::
+
+    mount.ceph 192.168.0.1:/ /mnt/mycephfs
+
+If there are multiple monitors, you can passes address separated by comma::
+
+   mount.ceph 192.168.0.1,192.168.0.2,192.168.0.3:/ /mnt/mycephfs
 
 If :doc:`ceph-mon <ceph-mon>`\(8) is running on a non-standard
 port::
 
-        mount.ceph monhost1:7000,monhost2:7000,monhost3:7000:/ /mnt/foo
+    mount.ceph 192.168.0.1:7000:/ /mnt/mycephfs
 
-To automatically determine the monitor addresses from local configuration::
+You can pass secret key as follows for CephX user as follows::
 
-        mount.ceph :/ /mnt/foo
+    mount.ceph /mnt/mycephfs -o name=fs,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
-To mount only part of the namespace::
+Or you can pass the secret into a file and pass it instead to avoid leaving
+secret key in shell's command history::
 
-        mount.ceph monhost1:/some/small/thing /mnt/thing
+    mount.ceph /mnt/mycephfs -o name=fs,secretfile=fs.secret
```

Comment:
I'd suggest changing the example "name" here to something like "fs_username" so that it's clear that "name" refers to a username instead of an alternate file system name.